jscomp/others/belt_Array.ml

```diff
@@ -408,6 +408,27 @@ let cmpU a b p =
 
 let cmp a b p = cmpU a b (fun[@bs] a b -> p a b)
 
+let partition f a =
+  let l = length a in
+  let i = ref 0 in
+  let j = ref 0 in
+  let a1 = makeUninitializedUnsafe l in
+  let a2 = makeUninitializedUnsafe l in
+  for ii = 0 to l - 1 do
+    let v = getUnsafe a ii in
+    if f v then (
```

Comment:
This introduced a currying in the loop, can you rename `partition` into `partitionU`  and call `f v [@bs]` here (you can check other examples)

Reply:
I also see I've messed up the order of the arguments. Guess it's intended to use with e.g. fast pipe. I've updated this and the currying now. Does it look OK?